meta-facebook/gt-cc/src/platform/plat_pldm_monitor.h

```diff
@@ -21,6 +21,7 @@
 extern uint8_t e1s_prsnt_pin[4][4];
 extern uint8_t nic_prsnt_pin[];
 extern uint8_t pex_sensor_num_table[];
+extern uint8_t e1s_sensor_table[16];
```

Comment:
Can you please define a macro for size 16 and use here and in plat_pldm_monitor.c

Reply:
I've change that into macro SSD_MAX_NUMBER, please check the update.